backend/src/main/java/heartbeat/client/GitHubFeignClient.java

```diff
@@ -14,19 +14,19 @@
 import java.util.List;
 
 @FeignClient(name = "githubFeignClient", url = "${github.url}")
-public interface GithubFeignClient {
+public interface GitHubFeignClient {
```

Comment:
Do I have design decoder for this client, you can seek help from guoqing

Reply:
Maybe do it in another card

Reply:
ok